src/containers/Storage/Disks/Disks.tsx

```diff
@@ -55,7 +57,7 @@ export function Disks({vDisks = [], viewContext}: DisksProps) {
             <div className={b('pdisks-wrapper')}>
                 {vDisks?.map((vDisk, index) => (
                     <PDiskItem
-                        key={vDisk?.PDisk?.StringifiedId}
+                        key={vDisk?.PDisk?.StringifiedId || index}
```

Comment:
There could be no proper id, use index in such case

Reply:
in which cases there may be no StringifiedId and why cant we always use index

I dont mean to rework anything - I just dont understand (:

Reply:
1. If there is no data about PDisk, then there would be no id, so we use index to prevent undefined ids - with undefined ids there is a bug where new disks added on every rerender
2. Generally it's not a good practice to use indexes as keys, since they could cause some unexpected behaviour if array is reordered or some elements added or deleted (and it's not recommended in react docs). So we use unique ids everywhere where it's possible. For the case of missing vdisks' pdisks there is no much difference, since they have no data and displayed the same way